var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -23,9 +23,10 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from spack.build_systems.cuda import CudaPackage
```

Comment:
If you rebase off of develop, you can remove this line now that #6746 has been merged.